app/assets/stylesheets/sequence_builder.css

```diff
@@ -209,8 +202,40 @@ span.prefix{
   font-weight: bold;
 }
 
-/* SITE-WIDE ISH */
+/* SITE-WIDE BACKGROUND */
 
 body.ng-scope {
 background: #EEE;
+}
+
+/* SITE WIDE STUFF */
+
+.green-button{
```

Comment:
I changed the old `green` class. You had the green class applied to both buttons and green sequence builder blocks. I think the styling should be different for blocks and buttons. I think I already cleaned it up from my last push of the sequence builder stuff.